Src/Support/Google.Apis.Auth/SignedTokenVerification.cs

```diff
@@ -49,7 +56,11 @@ internal static class SignedTokenVerification
             Task signatureVerificationTask = signedToken.Header.Algorithm switch
             {
                 "RS256" => VerifyRS256TokenAsync(signedToken, options, cancellationToken),
+#if NETSTANDARD2_0 || NET45
                 "ES256" => VerifyES256TokenAsync(signedToken, options, cancellationToken),
+#else
```

Comment:
Again, let's be really explicit on this - even more important than in the tests. We don't want to accidentally not support ES256 on netstandard2.1 if we ever add that as a target.